<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DryWidget</class>
 <widget class="QWidget" name="DryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>154</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Dry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widgetSpeed" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widgetTotalTime" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Dry Total Time(Min)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinDryTime">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>-2147483647</number>
                </property>
                <property name="maximum">
                 <number>2147483647</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Dry Speed</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinDrySpeed">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>-2147483647</number>
                </property>
                <property name="maximum">
                 <number>2147483647</number>
                </property>
                <property name="value">
                 <number>3000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Dry Time(max)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinDryMaxTime">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>-2147483647</number>
                </property>
                <property name="maximum">
                 <number>2147483647</number>
                </property>
                <property name="value">
                 <number>70</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>128</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
